classes/tools/parameter/analyzer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace mageekguy\atoum\tools\parameter;
+
+class analyzer
+{
+    public function getTypeHintString(\reflectionParameter $parameter): string
+    {
+        if (!$parameter->hasType()) {
+            return '';
+        }
+
+        $parameterType = $parameter->getType();
+
+        if ($parameterType instanceof \reflectionNamedType && $parameterType->getName() === 'mixed') {
+            // 'mixed' cannot be prefixed by nullable flag '?' and cannot be part of union types
```

Comment:
We do not usually add comments such this one.

Does it really bring any value here?

Reply:
I usually add in-code comments for specific cases. Can be removed if you want.

Reply:
let other contributor express their POV here.

IMHO, this comment is not necesary.